.github/scripts/status_update_common.py

```
@@ -0,0 +1,226 @@
+        
+        print("設定の初期化が完了しました")
+
+class GithubHandler:
+    def __init__(self, config: Config):
+        self.config = config
+        try:
+          self.github = Github(config.github_token)
+          self.repo = self.github.get_repo(config.github_repo)
+          self.issue_number = config.issue_number
+          self.issue = self.repo.get_issue(config.issue_number)
+        except Exception as e:
```

Ruff (0.8.2)

Redefinition of unused `e` from line 3

(F811)